core/src/components/datetime/datetime.tsx

```diff
@@ -141,6 +137,16 @@ export class Datetime implements ComponentInterface {
   @State() isPresented = false;
   @State() isTimePopoverOpen = false;
 
+  /**
+   * When non-null, will force the datetime to render the month
```

Comment:
```suggestion
   * When defined, will force the datetime to render the month
```

`forceRenderDate` doesn't use null anymore